functions/pupil-status/pupil-status-batch.service.js

```diff
@@ -93,8 +93,8 @@ function updatePupilStatuses (updates, context) {
     params.push({ name: `code${i}`, value: o.targetStatusCode, type: TYPES.NVarChar })
     params.push({ name: `pupilId${i}`, value: o.pupilId, type: TYPES.Int })
   })
-  context.log('updatePupilStatuses is attempting to execute the following sql:')
-  context.log(`${sql.join('\n')}`)
+  // context.log('updatePupilStatuses is attempting to execute the following sql:')
```

Comment:
you can delete this now, it was for debugging.

Reply:
done